### DISCUSSION TITLE
Invalid or negative modification date errors fix with AIO installation

### DISCUSSION THREAD

**Discussion by Charlie67j** [nextcloud/all-in-one#2306] [Questions] [answered]
I've searched for a conclusive answer for this one and I apologize if this has been covered. I'm getting the failure to sync on a couple of clients due to the invalid modification date error. I've found the official post about the way to fix this with the provided scripts. [link](https://github.com/nextcloud/desktop/wiki/How-to-fix-the-error-invalid-or-negative-modification-date) I'm curious if there is an official procedure for performing this fix on an AIO installation.

**Comment by szaimen** [comment, marked as answer]
Hi, basically go into the container with `sudo docker exec -it nextcloud-aio-nextcloud bash` and then follow the instructions at https://github.com/nextcloud/desktop/wiki/How-to-fix-the-error-invalid-or-negative-modification-date#files-stored-on-the-server. You'll find all needed values in the config.php file, located at /var/www/html/config/config.php inside the container.

**Reply by Charlie67j** [reply]
Thanks! I'll give that a go.

**Reply by MichaKersloot** [reply]
Sorry doesn't work because the container uses the bussy-box implementation of 'find' and the script uses incompatible parameters with that.

```
find: unrecognized: -newermt
BusyBox v1.35.0 (2022-08-01 15:14:44 UTC) multi-call binary.

Usage: find [-HL] [PATH]... [OPTIONS] [ACTIONS]
```

**Reply by szaimen** [reply]
You could probably add find into the container with `apk add find`

**Reply by MichaKersloot** [reply]
that would be 'apk add findutils' . But the script doesn't find any problems. I'll keep digging.
